src/selector.rs

```diff
@@ -953,6 +958,17 @@ pub enum PseudoElement<'i> {
     /// A part name selector.
     part: ViewTransitionPartSelector<'i>,
   },
+  /// The [::picker](https://drafts.csswg.org/css-forms-1/#the-picker-pseudo-element) pseudo element.
```

Comment:
Is this actually supported? The spec seems to suggest that only the functional form is allowed.

Reply:
Only the functional form works for now, so I’ll remove the non-functional form to avoid confusion. I originally added support for it because the spec suggests it may be supported in the future. 😬
![CleanShot 2025-04-22 at 11 33 26@2x](https://github.com/user-attachments/assets/9b02739a-41ca-4aea-8f21-5378aa25f0d8)

Reply:
Removed the non-functional form and added tests for `::picker`. Thanks! https://github.com/parcel-bundler/lightningcss/pull/957/commits/afac67a3b3a03d26878d036e51a38d4e2c94072d